src/OpeningHours.php

```diff
@@ -150,13 +151,17 @@ public function isClosed(): bool
         return $this->isClosedAt(new DateTime());
     }
 
-    public function nextOpen(DateTimeInterface $dateTime): DateTime
+    public function nextOpen(DateTimeInterface $dateTime): DateTimeInterface
```

Comment:
Unfortunately changing the return type is a breaking change.

Reply:
Nop because your PHP requirement: `php: ^7.0` implies that any `DateTime` object implements `DateTimeInterface`, so unless using reflexion, there is no break, but with reflexion, anything is breaking change.